src/components/LogRow.vue

```diff
@@ -45,6 +45,15 @@
   grid-template-columns: 84px minmax(0, 1fr) 150px;
 }
 
+@media (max-width: 768px) {
+  .log-row { @apply
+  grid
+  py-2;
+  grid-template-columns: none;
+  grid-template-rows: 20px minmax(0, 1fr) 50px;
+}
+}
+
```

Comment:
Thoughts on this instead?
```css
@media (max-width: 768px) {
  .log-row {
    grid-template-areas: 
      "leading trailing"
      "content content";
    grid-template-columns: 1fr 1fr;
    grid-template-rows: auto auto;
  }

  .log-row__leading { 
    grid-area: leading;
  }

  .log-row__trailing { 
    grid-area: trailing;
  }

  .log-row__content { 
    grid-area: content;
  }
}

```

which if I free-handed this correctly will mean we'd have:
[ level ] [ timestamp ]
[           content         ]

instead of:
[ level ]
[ content ]
_______ [ timestamp ]

Reply:
and for even smaller screens:

```css
@media (max-width: 425px) {
  .log-row {
    grid-template-areas: 
      "leading"
      "trailing"
      "content";
    grid-template-columns: 1fr;
    grid-template-rows: auto auto auto;
  }

  .log-row__trailing { @apply
    text-left
  }
}
```

Reply:
I think it needs a bit of tweaking to get the log level button at the same height as the time and trailing info:

<img width="328" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/40272060/975f1abf-36cf-452d-9175-80b24166df60">

Going to merge as is right now and then work on getting your suggestion to fit.